src/benchmarklib/benchmark_runner.hpp

```diff
@@ -31,9 +33,8 @@ class SQLiteWrapper;
 
 class BenchmarkRunner {
```

Comment:
Optional, maybe it's time to give this class a one-liner doc comment?

Reply:
```
// The BenchmarkRunner is the main class for the benchmark framework. It gets initialized by the benchmark binaries
// (e.g., tpch_benchmark.cpp). They then hand over the control to the BenchmarkRunner (inversion of control), which
// calls the supplied table generator, runs and times the benchmark items, and reports the benchmark results.
```
?

Reply:
great. dunno if "inversion of control" helps me, but maybe it helps others?

Reply:
Add "(aka. Hollywood Principle)"?

Reply:
nope, didn't ring a bell for me, prior to googling it.